src/components/Dropdown/__snapshots__/DropdownList.test.js.snap

```diff
@@ -34,7 +34,7 @@ exports[`Archives | Components | FiltersMenu renders correctly 1`] = `
       />
     }
     isDisabled={false}
-    isFocused={false}
+    isFocused={true}
```

Comment:
Ok, so since now dropdown is default focused?
btw `Archives | Components | FiltersMenu` this test should have another name ;)
and we should avoid `renders correctly` tests

Reply:
it's related to autofocus feature, this is the first available dropdown list item (not disabled)